packages/wallets/src/features/cfd/screens/ChangePassword/MT5ChangePasswordScreens.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useState } from 'react';
+import { SentEmailContent } from '../../../../components';
+import { Tab, Tabs, WalletText } from '../../../../components/Base';
+import IcBackArrow from '../../../../public/images/ic-back-arrow.svg';
+import { TPlatforms } from '../../../../types';
+import MT5ChangeInvestorPasswordScreens from './InvestorPassword/MT5ChangeInvestorPasswordScreens';
+import TradingPlatformChangePasswordScreens from './TradingPlatformChangePasswordScreens';
+
+type TProps = {
+    platform: TPlatforms.All;
+    title: string;
+};
+
+const MT5ChangePasswordScreens: React.FC<TProps> = ({ platform, title }) => {
+    const [showSentEmailContentWithoutTabs, setShowSentEmailContentWithoutTabs] = useState(false);
+    const [tabNumber, setTabNumber] = useState(0);
+
+    return showSentEmailContentWithoutTabs ? (
+        <Fragment>
```

Comment:
I don't think we need the fragment for this because it is enclosed within a `div`

Reply:
@aizad-deriv there are 2 divs in this block